packages/expansion-hub/src/errors/gpio-errors.ts

```diff
@@ -0,0 +1,35 @@
+import {NackError, setPrototypeOf} from "./NackError.js";
+
+export class GpioNotConfiguredForOutputError extends NackError {
+    gpioIndex: number;
```

Comment:
Let's rename to `digitalChannel` and rename the class to `DigitalChannelNotConfiguredForOutputError`. The error message also needs updating.

This applies to everything else in this file as well, and the name of the file itself.